native/app/components/ListPanel.js

```diff
@@ -23,13 +24,28 @@ class ListPanel extends React.Component {
     this._onRefresh = () => {
       this.setState({ refreshing: true });
 
-      return sync.loadFromKinto(kintoClient, props.state.sync.loginDetails).then(() => {
-        this.setState({ refreshing: false });
-        this.setState({ snackbarSyncedvisible: true })
+      browser.runtime.sendMessage({
+        action: KINTO_LOADED
       });
     }
   }
 
+  componentWillReceiveProps(newProps) {
+    if (!newProps.state.sync.isSyncing) {
+      // We do not display snackbar on sync if triggered from other than pullRefresh
```

Comment:
What happens if we change this and always show the snackbar, no matter how you got the new notes?

Reply:
It looks weird 😅, but open to any suggestion.
![snackbaroverdrawer](https://user-images.githubusercontent.com/3018618/39105211-d6ff804e-46de-11e8-96f6-43f936328036.gif)

Reply:
> Nothing wrong, but feel to me like a bit intrusive ... what do you think ?

To clarify, we only show the Snackbar if the user is on the ListPanel view. In the single note view we have the header text to show the status i guess

Reply:
I like your suggestions and will apply right away ! 😁.

Reply:
`elevation` might not be the right thing to stack the Snackbar behind the drawer. Glancing at the pr it's not clear to me at this time how the thing shows up above the Drawer in our case. But we might not have to worry about that because we can time it such that we close the Drawer first and then show the snackbar.

(ref: https://github.com/callstack/react-native-paper/pull/305/files )